test/testinput/bufr_sfcshp.yaml

```diff
@@ -46,7 +46,7 @@ observations:
       obsdataout: "./testrun/bufr_sfcshp.nc"
 
       globals:
-        - name: "dataType@MetaData"
+        - name: "platformLongDescription"
```

Comment:
@emilyhcliu this was adding a key `dataType@MetaData` to the global attributes, not adding anything to the `MetaData` group

in the [conventions](https://github.com/JCSDA-internal/ioda/blob/b27373f084f63d7565338f13588118a73787890d/share/ioda/yaml/validation/ObsSpace.yaml#L52) there are many options `long_name`, etc chose this one as think it is appropriate.  Confident this is not being used for anything so do like this name and have been trying to use it in other converters